src/new-components/select-field/select-field.test.tsx

```diff
@@ -63,6 +63,11 @@ describe('SelectField', () => {
         )
     })
 
+    it('is marked as invalid and with an error message if an error message is given', () => {
+        render(<SelectField data-testid="select-field" label="Gender" error="Must select one" />)
```

Comment:
Could we maybe change this label or error message?

```suggestion
        render(<SelectField data-testid="select-field" label="Favourite ice cream flavour" error="Must select one" />)
```